hubblestack/utils/signing.py

```diff
@@ -194,32 +256,48 @@ def __init__(self, public_crt, ca_crt):
         self.trusted = list()
         # NOTE: trusted is mostly useless. We do use it in
         # testing, and that's probably about it
-
+        seconds_day = 86400
         already = set()
         for i in read_certs(ca_crt):
+            log_level = log.debug
             digest = i.digest('sha1')
             if digest in already:
                 continue
             already.add(digest)
             digest = digest.decode() + " " + stringify_ossl_cert(i)
-            log.debug('adding %s as a trusted certificate approver', digest)
             self.store.add_cert(i)
             self.trusted.append(digest)
+            log_level = log.debug
+            if check_verif_timestamp(digest, dampener_limit=seconds_day) == True:
+                log_level = log.splunk
+            status = STATUS.VERIFIED
+            str_ca = stringify_cert_files(ca_crt)
+            msg = 'ca cert | file: "{}" | status: {} | digest "{}" | added to verify store'
+            log_level(msg.format(str_ca, status, digest))
 
         for i in read_certs(*untrusted_crt):
             digest = i.digest('sha1')
             if digest in already:
                 continue
             already.add(digest)
             digest = digest.decode() + " " + stringify_ossl_cert(i)
-            log.debug('checking to see if %s is trustworthy', digest)
             try:
                 ossl.X509StoreContext(self.store, i).verify_certificate()
                 self.store.add_cert(i)
                 self.trusted.append(digest)
-                log.debug('  added to verify store')
+                status = STATUS.VERIFIED 
+                log_level = log.debug
             except ossl.X509StoreContextError as exception_object:
-                log.debug('  not trustworthy: %s', exception_object)
+                # log at either log.error or log.critical according to the error code
+                status = STATUS.FAIL
+                pass
+            if check_verif_timestamp(digest, dampener_limit=seconds_day) == True:
```

Comment:
Redundant comparison to True, check_verif_timestamp() returns bool value already.